### PR TITLE
Fix POST parameter name

### PR DESCRIPTION
**Pull request by eugenefab**
In the `create` method the POST parameter for the environment variable name is currently incorrect. It's changed to the value described in the API documentation: `name`.

### CODE REVIEW

**Review by xginn8 — APPROVED** [review]
Thanks for contributing @eugenefab! This change lgtm. It would be good to validate these end-to-end by perhaps spinning up the API from `open-balena` or something, but that's outside the scope of this change. I want to wait for @nghiant2710's feedback before merging since the tests are broken and I am not sure how external contributions will fit into our versioning system.

**Comment by eugenefab** [comment]
> Thanks for contributing @eugenefab! This change lgtm. It would be good to validate these end-to-end by perhaps spinning up the API from `open-balena` or something, but that's outside the scope of this change. I want to wait for @nghiant2710's feedback before merging since the tests are broken and I am not sure how external contributions will fit into our versioning system.

@xginn8 Yea, I was trying to see why the tests were failing and how to fix them, but I'm not seeing why this change would cause a failure. Seems almost like a test configuration issue. If it helps, I just deployed a fork of this SDK with this change in it, to our own integration environment and the API calls are succeeding now.

Thanks for looking at this so quickly!

**Comment by nghiant2710** [comment]
@eugenefab thank you for the PR, the tests failed since our CI can't pull that branch from your fork. LGTM but I will need to do a review if there are other API changes that break the SDK so I will merge this to my working branch and will release really soon.

**Pull request merged**
